source/1_engine/rex_engine/include/rex_engine/filesystem/vfs.h

```diff
@@ -0,0 +1,70 @@
+#pragma once
```

Comment:
Going to put this here, but it might be an idea to use a "Path" structure ( even if it is a using for rsl::string_view ) this just give a better overview of what is expected as input for the VFS.

Reply:
Yes, I want to use a path structure as well, it'll be using a string underneath though, not a string view

Reply:
Made an issue for this